Kudu.Core/SiteExtensions/FeedExtensions.cs

```diff
@@ -103,6 +103,9 @@ public static class FeedExtensions
                         FileSystemHelpers.EnsureDirectory(Path.GetDirectoryName(fullPath));
                         using (Stream writeStream = FileSystemHelpers.OpenWrite(fullPath))
                         {
+                            // reset length of file stream
+                            writeStream.SetLength(0);
```

Comment:
why [line 107](https://github.com/projectkudu/kudu/pull/1637/files#diff-aeb25288ddaac13782bf8b797f7a288cR107)  and [line 192](https://github.com/projectkudu/kudu/pull/1637/files#diff-aeb25288ddaac13782bf8b797f7a288cR192) are necessary?